msbuild/Xamarin.Localization.MSBuild/MSBStrings.resx

```diff
@@ -834,10 +834,6 @@
         </value>
     </data>
 
-    <data name="E0176" xml:space="preserve">
-        <value>The provided Archive Path is not from a valid archive file (.xcarchive). Archive Path: '{0}'</value>
-    </data>
-    
```

Comment:
minor: `W0176` should be moved here to keep the order correct.

Otherwise someone, who does not look up, might use `0177` again for the next error they add.